hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/fullConfig-applicationContext-hazelcast.xml

```diff
@@ -1029,6 +1029,7 @@
                 <hz:persistence-enabled>false</hz:persistence-enabled>
                 <hz:base-dir>/custom-dir</hz:base-dir>
                 <hz:data-load-timeout-seconds>30</hz:data-load-timeout-seconds>
+                <hz:cp-member-priority>-1</hz:cp-member-priority>
```

Comment:
Are negative values allowed intentionally?

Reply:
In theory yes, the default value for priorities is 0. If someone decides to make one CP member a non-priority, the easiest way would be to set a negative priority (eg -1).

Reply:
Thanks for the review!